webapp/src/Routes.js

```diff
@@ -89,6 +90,33 @@ export default class Routes extends React.Component {
           path={this.addLegacySupport(locations.parcelMapDetail())}
           component={AtlasPage}
         />
+        {isFeatureEnabled('BIDS') && (
```

Comment:
This is OK but maybe we can add `{/*Bids*/}` and group this routes with the estates ones as well?
I think it'll look a bit more organised because parcels and estates already have a ton of routes, what do you think?